hw/ip/padctrl/padring.core

```diff
@@ -0,0 +1,65 @@
+CAPI=2:
```

Comment:
This file shouldn't really be part of this PR.

Reply:
It's needed such that the corresponding regression config works. I can split it out into its own PR to make this commit clean.